charts/hyperflow-engine/templates/deployment.yml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hyperflow-engine
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      name: hyperflow-engine
+      component: hyperflow-engine
+  template:
+    metadata:
+      labels:
+        name: hyperflow-engine
+        component: hyperflow-engine
+    spec:
+      containers:
+        - name: hyperflow
+          image: {{ .Values.containers.hyperflow.image }}
+          imagePullPolicy: Always
+          {{- with .Values.resources }}
+          resources: {{ . | toYaml | nindent 10 }}
+          {{- end }}
+          env:
+          - name: REDIS_URL
+            # Change that value to reflect your and namespace and cluster FQDN
+            value: redis://redis.default.svc.cluster.local:6379
```

Comment:
this should not be hardcoded! applies to a pull request as a whole
please use templates like those ones for service generation URL:
https://github.com/onedata/charts/blob/master/stable/onedata-chart-utils/templates/_helpers.tpl#L30-L38